### PR TITLE
test(selenium): Retry when opening chrome fails

### PR DESCRIPTION
**Pull request by billyvg**
Use `TimedRetryPolicy` to retry opening Chrome when it fails. Also add `self-contained-html` option when running `pytest` so the py test results can be uploaded to zeus.

### CODE REVIEW

File: src/sentry/utils/pytest/selenium.py [hunk 1]
```diff
@@ -312,6 +312,21 @@ def percy(request):
...
 
+start_chrome_retry_attempts = 5
+
+
+def start_chrome(**chrome_args):
+    try:
+        return webdriver.Chrome(**chrome_args)
+    except WebDriverException as e:
+        global start_chrome_retry_attempts
+        if start_chrome_retry_attempts > 0:
```
Comment (EvanPurkhiser) [inline src/sentry/utils/pytest/selenium.py]: could use https://github.com/getsentry/sentry/blob/master/src/sentry/utils/retries.py#L28 if you wanted

Comment (billyvg) [reply src/sentry/utils/pytest/selenium.py]: Good call

File: src/sentry/utils/retries.py [hunk 2]
```diff
@@ -97,6 +95,8 @@ def __call__(self, function):
                         self.clock.sleep(delay)
         finally:
             if self.metric_instance:
+                from sentry.utils import metrics
```
Comment (billyvg) [inline src/sentry/utils/retries.py]: If we don't move this we get errors:

```
sentry git/test/acceptance/test-chrome-retry-idk
(sentry) ❯ py.test tests/acceptance/test_organization_global_selection_header.py
Traceback (most recent call last):
  File "/Users/billy/.ve/sentry/bin/py.test", line 10, in <module>
    sys.exit(main())
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 63, in main
    config = _prepareconfig(args, plugins)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 207, in _prepareconfig
    pluginmanager=pluginmanager, args=args
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/manager.py", line 92, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/manager.py", line 86, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/callers.py", line 203, in _multicall
    gen.send(outcome)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/helpconfig.py", line 94, in pytest_cmdline_parse
    config = outcome.get_result()
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/callers.py", line 81, in get_result
    _reraise(*ex)  # noqa
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 687, in pytest_cmdline_parse
    self.parse(args)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 895, in parse
    self._preparse(args, addopts=addopts)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 851, in _preparse
    early_config=self, args=args, parser=self._parser
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/manager.py", line 92, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/manager.py", line 86, in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/callers.py", line 208, in _multicall
    return outcome.get_result()
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/callers.py", line 81, in get_result
    _reraise(*ex)  # noqa
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 750, in pytest_load_initial_conftests
    self.pluginmanager._set_initial_conftests(early_config.known_args_namespace)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 383, in _set_initial_conftests
    self._try_load_conftest(anchor)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 389, in _try_load_conftest
    self._getconftestmodules(anchor)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 424, in _getconftestmodules
    mod = self._importconftest(conftestpath.realpath())
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 474, in _importconftest
    self.consider_conftest(mod)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 527, in consider_conftest
    self.register(conftestmodule, name=conftestmodule.__file__)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 328, in register
    self.consider_module(plugin)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 533, in consider_module
    self._import_plugin_specs(getattr(mod, "pytest_plugins", []))
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 538, in _import_plugin_specs
    self.import_plugin(import_spec)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 586, in import_plugin
    self.register(mod, modname)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 328, in register
    self.consider_module(plugin)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 533, in consider_module
    self._import_plugin_specs(getattr(mod, "pytest_plugins", []))
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 538, in _import_plugin_specs
    self.import_plugin(import_spec)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/config/__init__.py", line 565, in import_plugin
    __import__(importspec)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/_pytest/assertion/rewrite.py", line 304, in load_module
    exec(co, mod.__dict__)
  File "/Users/billy/Dev/sentry/src/sentry/utils/pytest/selenium.py", line 20, in <module>
    from sentry.utils.retries import TimedRetryPolicy
  File "/Users/billy/Dev/sentry/src/sentry/utils/retries.py", line 11, in <module>
    from sentry.utils import metrics
  File "/Users/billy/Dev/sentry/src/sentry/utils/metrics.py", line 16, in <module>
    metrics_skip_all_internal = getattr(settings, "SENTRY_METRICS_SKIP_ALL_INTERNAL", False)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/django/conf/__init__.py", line 55, in __getattr__
    self._setup(name)
  File "/Users/billy/.ve/sentry/lib/python2.7/site-packages/django/conf/__init__.py", line 41, in _setup
    % (desc, ENVIRONMENT_VARIABLE))
django.core.exceptions.ImproperlyConfigured: Requested setting SENTRY_METRICS_SKIP_ALL_INTERNAL, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
```

**Review by wedamija — APPROVED** [review]

**Review by joshuarli — APPROVED** [review]
‾\_(ツ)_/‾

**Review by markstory — APPROVED** [review]

**Pull request merged**